src/components/MaplibreMap.jsx

```diff
@@ -236,10 +236,7 @@ const MaplibreMap = ({ view, setView }) => {
 
       if (clickedFeature.layer.id === 'sites-unclustered-layer') {
         const { sample_event_id } = clickedFeature.properties
-        const newQueryParams = getURLParamsSnapshot()
-        newQueryParams.set('sample_event_id', sample_event_id)
-        updateURLParams(newQueryParams)
-        setSelectedMarkerId(sample_event_id)
+        updateCurrentSampleEvent(sample_event_id)
```

Comment:
turned this into a reusable function so we can use it in the metrics pane as well.